src/store/validation/index.ts

```diff
@@ -63,15 +64,16 @@ export const getters: GetterTree<IValidationState, IRootState> = {
       })
     })
     return accumulator
-  }
+  },
+  validationStatuses: state => state.validationStatuses,
```

Comment:
Wondering why would we need to create the exact same name of state here? We can avoid this by mapping State instead of Getter here https://github.com/FLOIP/flow-builder/pull/88/files#diff-49480d9a22d67cf509e7509839b7497a30f7b37674851a33cd2b6a343bd7fb68R85 I think

Reply:
I was under the belief that accessing the state directly was not a good practice, hence created a getter. I might have the wrong understanding, I removed it.